core/state_transition.go

```diff
@@ -23,12 +23,16 @@ import (
 	"math/big"
 
 	"github.com/XinFinOrg/XDPoSChain/common"
+	cmath "github.com/XinFinOrg/XDPoSChain/common/math"
 	"github.com/XinFinOrg/XDPoSChain/core/types"
 	"github.com/XinFinOrg/XDPoSChain/core/vm"
+	"github.com/XinFinOrg/XDPoSChain/crypto"
 	"github.com/XinFinOrg/XDPoSChain/log"
 	"github.com/XinFinOrg/XDPoSChain/params"
 )
 
+var emptyCodeHash = crypto.Keccak256Hash(nil)
```

Comment:
would const be better here

Reply:
But Golang's grammar does not allow const here. Don't worry, this value is only calculated once when init package.

Reply:
> would const be better here?

Golang's grammar does not allow const here. This PR #685 will solve your concerns.